Example/ViewController.swift

```diff
@@ -22,6 +23,39 @@ class ViewController: UITableViewController {
         }
     }
 
+    @IBAction func actionShowMiniAppById() {
+        let alert = UIAlertController(title: "Please enter Mini App ID",
```

Comment:
Looks like a long method, on the Android side we get complaints about something over 15 lines.
And, possibly extracting the messages into a single place. It helps to keep things tidy. What do you think?

Reply:
mmm well, it's the shortest way to create such a dialog. I could extract several methods from this, or make an extension if you think it's valuable for a demo  
As for the messages, we could use localizable strings to do things properly, what do you think about this? By the way this makes code a little bit more verbose

Reply:
Overall, it helps to keep the code tidy. Of course, for the demo app sometimes we too make some trade-offs.

I think a minor refactor (usually less than <20 mins) is good in general, the next person might feel glad to have found some pieces in place when doing similar stuff. What do you think?
btw, localizable strings sound good to me.